docs/source/cli_help/status.rst

```diff
@@ -79,3 +79,29 @@ With ``--color=always``, git machete always emits colors and with ``--color=auto
 -L, --list-commits-with-hashes    Additionally list the short hashes and messages of commits introduced on each branch.
 
 --no-detect-squash-merges         Only consider *strict* (fast-forward or 2-parent) merges, rather than rebase/squash merges, when detecting if a branch is merged into its upstream (parent).
+
+ .. note::
+
+    To make it easier to select branch name from the ``status's`` output on certain terminals (E.g. Alacritty), you can add additional
+    space between ``└─`` and ``branch name`` by setting git config key ``machete.status.extraSpaceBeforeBranchName`` = ``true``.
+
+    E.g.
+        default:
+
+        .. code-block:: git
+
+          develop
+          │
+          ├─feature_branch1
+          │
+          └─feature_branch2
```

Comment:
Super-pedantic-nitpicking-warning!

This is indented 2 spaces, code below is indented 3 — 3 is often used for ReST, multiples of 2 are used often elsewhere, so both are fine… but I think it should be one of them, not both

Reply:
🕵🏻